File: Homeworks/C#2/MethodsHomework/03.EnglishDigit/EnglishDigit.cs
/*
Problem 3. English digit

Write a method that returns the last digit of given integer as an English word.
Examples:

input	output
512	    two
1024	four
12309	nine
*/
using System;

class EnglishDigit
{                                                                               
    static string EnglishWord(int number)                                       
    {                                                                           
        int digit = number % 10;                                                
                                                                                
                                                                                
        switch (digit)                                                          
        {
            case 0: 
                Console.WriteLine("zero"); 
                break;
            case 1:
                Console.WriteLine("one");
                break;
            case 2:
                Console.WriteLine("two");
                break;
            case 3:
                Console.WriteLine("three");
                break;
            case 4:
                Console.WriteLine("four");
                break;
            case 5:
                Console.WriteLine("five");
                break;
            case 6:
                Console.WriteLine("six");
                break;
            case 7:
                Console.WriteLine("seven");
                break;
            case 8:
                Console.WriteLine("eight");
                break;
            case 9:
                Console.WriteLine("nine");
                break;
        }
        return "";
    }
        

    static void Main()
    {
        Console.Write("Please, enter number: ");
        int number = int.Parse(Console.ReadLine());
        Console.Write("Last digit of {0} as an english word is: ", number); 
        EnglishWord(number);
    }

}

File: Homeworks/JavaScriptFundamentals/06.Functions/01.EnglishDigit/EnglishDigit.js
/*
 Write a function that returns the last digit of given integer as an English word.
 Examples: 512 ? "two", 1024 ? "four", 12309 ? "nine".
 */
var jsConsole,
    button = document.getElementById('check'),
    lastDigit,
    inputNumber;

button.onclick = digitAsWord;

function digitAsWord() {
    inputNumber = jsConsole.readInteger("#number");

    lastDigit = inputNumber % 10;

    switch(lastDigit) {
        case 0:
            jsConsole.writeLine('The last digit is Zero');
            console.log('The last digit is Zero');
            break;
        case 1:
            jsConsole.writeLine('The last digit is One');
            console.log('The last digit is One');
            break;
        case 2:
            jsConsole.writeLine('The last digit is Two');
            console.log('The last digit is Two');
            break;
        case 3:
            jsConsole.writeLine('The last digit is Three');
            console.log('The last digit is Three');
            break;
        case 4:
            jsConsole.writeLine('The last digit is Four');
            console.log('The last digit is Four');
            break;
        case 5:
            jsConsole.writeLine('The last digit is Five');
            console.log('The last digit is Five');
            break;
        case 6:
            jsConsole.writeLine('The last digit is Six');
            console.log('The last digit is Six');
            break;
        case 7:
            jsConsole.writeLine('The last digit is Seven');
            console.log('The last digit is Seven');
            break;
        case 8:
            jsConsole.writeLine('The last digit is Eight');
            console.log('The last digit is Eight');
            break;
        case 9:
            jsConsole.writeLine('The last digit is Nine');
            console.log('The last digit is Nine');
            break;
    }
}